### DISCUSSION TITLE
Questions about custom fonts in Wasm and Desktop

### DISCUSSION THREAD

**Discussion by luckypong** [unoplatform/uno#17236] [Q&A] [answered]
Questions about password display characters not being supported and additional questions when applying Korean fonts (migrating from a project-specific platform to a single-project platform)

Question 1) ● PasswordBox is not displayed in Wasm. There is no problem with previous Skia, mobile platforms, and current single project desktop platforms (#[eaded92](https://github.com/jeromelaban/Uno/commit/eaded92131980ef7388f7fef7f5f2927bc52ac77)).

Windows
![스크린샷 2024-06-20 102843](https://github.com/unoplatform/uno/assets/11326846/c82cdd2f-05c6-49c0-9ba1-c57f28d40324)

Wasm
![스크린샷 2024-06-20 103522](https://github.com/unoplatform/uno/assets/11326846/5b0e61b9-920b-4298-be82-22b3325ea3f6)

Desktop
![스크린샷 2024-06-20 103115](https://github.com/unoplatform/uno/assets/11326846/40ae64c3-9ad8-41b8-ba3d-094b5b0e9c68)

Question 2) How to use Resizetiezer UnoImage and deploy svg at the same time? (using svg)

Question 3) How to change the dialog in ShowMessageDialogAsync to ContentDialog style only in Windows (WinUI)?
[As-Is]
![스크린샷 2024-06-20 104842](https://github.com/unoplatform/uno/assets/11326846/ef35fed5-a68e-43ad-9ea3-375500e239f2)
[To-Be]
![스크린샷 2024-06-20 104935](https://github.com/unoplatform/uno/assets/11326846/9f0f4d0f-3b4c-4960-a212-1e0855e27db6)

Sample Solution)
[UnoApp1.zip](https://github.com/user-attachments/files/15907411/UnoApp1.zip)

**Comment by jeromelaban** [comment, marked as answer]
For the first issue, let's track in https://github.com/unoplatform/uno/issues/17246

**Reply by jeromelaban** [reply]
The other questions are in  https://github.com/unoplatform/uno/discussions/17250 and https://github.com/unoplatform/uno/discussions/17249.
